includes/incoming-events/class-woocommerce-membership-updated.php

```diff
@@ -77,13 +77,13 @@ public function update_membership() {
 		$user_membership = wc_memberships_get_user_membership( $user->ID, $local_plan_id );
 
 		if ( null === $user_membership ) {
+			// Create a new membership if it doesn't exist and link it to the remote membership.
 			$user_membership = wc_memberships_create_user_membership(
 				[
 					'plan_id' => $local_plan_id,
 					'user_id' => $user->ID,
 				]
 			);
-
 			update_post_meta( $user_membership->get_id(), Memberships_Admin::NETWORK_MANAGED_META_KEY, true );
```

Comment:
I mean you can delete these 3 lines

Reply:
Ah whoops! Misread that. Updated in https://github.com/Automattic/newspack-network/pull/192/commits/b0678ebd08c4685b06e11417cd93838bd64a9f5f

Reply:
Sorry. Also snuck in a change to one of the comments relevant here: https://github.com/Automattic/newspack-network/pull/192/commits/299a9ca153bfa95eedc376e71a0872bfa797fe01